diagnostic.md

```diff
@@ -7,27 +7,32 @@ Write your answers inside this file, where it's indicated by the comments.
     repository in your answer.
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    git clone git@github.com:byau/git-basics-diagnostic.git
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
     ```sh
-    <!-- Remove this comment and place your answer here. -->
+    cd git-basics-diagnostic
```

Comment:
create a branch and move into it ?

git checkout -b <branchname> ?  let me know if that’s what you are looking for?

> On Jan 20, 2016, at 9:46 AM, Matt Brendzel <notifications@github.com> wrote:
> 
> In diagnostic.md <https://github.com/ga-wdi-boston/git-basics-diagnostic/pull/45#discussion_r50261987>:
> 
> >      ```
> >  
> >  2.  What do you do after cloning a repository, but before starting work?
> >  
> >      ```sh
> > -    <!-- Remove this comment and place your answer here. -->
> > +    cd git-basics-diagnostic
> I suppose that is technically true, but it's not what we were looking for. What should you do before starting to make your own changes?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ga-wdi-boston/git-basics-diagnostic/pull/45/files#r50261987>.
>